libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -49,8 +49,8 @@
 #include <AP_Logger/AP_Logger.h>
 
 #define AP_ARMING_COMPASS_MAGFIELD_EXPECTED 530
-#define AP_ARMING_COMPASS_MAGFIELD_MIN  185     // 0.35 * 530 milligauss
-#define AP_ARMING_COMPASS_MAGFIELD_MAX  875     // 1.65 * 530 milligauss
+#define AP_ARMING_COMPASS_MAGFIELD_MIN  200     // 0.35 * 530 milligauss, limited further for mttr ops
+#define AP_ARMING_COMPASS_MAGFIELD_MAX  700     // 1.65 * 530 milligauss, limited further for mttr ops
```

Comment:
Please update the math in the comment to show source of new value

Reply:
done